ahnlich/client/src/lib.rs

```diff
@@ -56,6 +71,37 @@
 //! );
 //! let results = pipeline.exec().await.unwrap();
 //! ```
+//!
+//! ### AI Client
+//! ```rust
+//! use ahnlich_client_rs::ai::AIClient;
+//! use ahnlich_client_rs::prelude::*;
+//! use std::collections::HashSet;
+//!
+//! let ai_client = AIClient::new("127.0.0.1".into(), 1369).await;
```

Comment:
Unwrap after await cuz this is a result... please help change for DBClient cuz I made the same mistake